runtime/src/registry.rs

```diff
@@ -178,6 +178,28 @@ decl_module! {
             Ok(())
         }
 
+        #[weight = SimpleDispatchInfo::InsecureFreeNormal]
+        pub fn register_member(origin, message: message::RegisterMember) -> DispatchResult {
+            let sender = ensure_signed(origin)?;
+
+            let org = store::Orgs::get(message.org_id.clone()).ok_or(RegistryError::InexistentOrg)?;
+            if !org_has_member_with_account(&org, sender) {
+                return Err(RegistryError::InsufficientSenderPermissions.into());
+            }
+
+            if None == store::Users::get(message.user_id.clone()) {
+                return Err(RegistryError::InexistentUser.into());
+            }
+
+            if org.members.contains(&message.user_id) {
+                return Err(RegistryError::DuplicateMember.into());
+            }
+
+            store::Orgs::insert(message.org_id.clone(), org.add_member(message.user_id.clone()));
```

Comment:
We mostly try to separate such lines, like so:

```suggestion
            let org_with_member = org.add_member(message.user_id.clone());
            store::Orgs::insert(message.org_id.clone(), org_with_member);
```